packages/language-server/src/__test__/completion.test.ts

```diff
@@ -1190,6 +1252,303 @@ suite('Completions', function () {
         },
       })
     })
+    test('@@index(fields: [|])', () => {
+      assertCompletion({
+        schema: /* Prisma */ `
+          model ThirdUser {
+              firstName String
+              lastName String
+              isAdmin Boolean @default(false)
+              @@index(field: [|])
+          }`,
+        expected: {
+          isIncomplete: false,
+          items: [
+            { label: 'firstName', kind: CompletionItemKind.Field },
+            { label: 'lastName', kind: CompletionItemKind.Field },
+            { label: 'isAdmin', kind: CompletionItemKind.Field },
+          ],
+        },
+      })
+    })
+    test('MongoDB: @@index([|])', () => {
+      assertCompletion({
+        provider: 'mongodb',
+        schema: /* Prisma */ `
+          type Address {
+            street String
+            number Int
+          }
+          model User {
+            id      Int     @id
+            email   String
+            address Address
+            @@index([|])
+          }`,
+        expected: {
+          isIncomplete: false,
+          items: [
+            { label: 'id', kind: CompletionItemKind.Field },
+            { label: 'email', kind: CompletionItemKind.Field },
+            { label: 'address', kind: CompletionItemKind.Field },
+          ],
+        },
+      })
+    })
+    test('MongoDB: @@index(ad[|])', () => {
+      assertCompletion({
+        provider: 'mongodb',
+        schema: /* Prisma */ `
+          type Address {
+            street String
+            number Int
+          }
+          model User {
+            id      Int     @id
+            email   String
+            address Address
+            @@index([a|])
+          }`,
+        expected: {
+          isIncomplete: false,
+          items: [
+            { label: 'id', kind: CompletionItemKind.Field },
+            { label: 'email', kind: CompletionItemKind.Field },
+            { label: 'address', kind: CompletionItemKind.Field },
```

Comment:
Should this be only
```suggestion
            { label: 'address', kind: CompletionItemKind.Field },
```
?

Reply:
Yes it would be better but this would be a new behavior, out of scope here